packages/devextreme/js/__internal/ui/chat/messagegroup.ts

```diff
@@ -24,9 +24,13 @@ export type MessageGroupAlignment = 'start' | 'end';
 export interface Properties extends WidgetOptions<MessageGroup> {
   items: Message[];
   alignment: MessageGroupAlignment;
+  // eslint-disable-next-line
+  messageTemplate: any;
 }
 
 class MessageGroup extends Widget<Properties> {
+  private _messageBubbles?: MessageBubble[];
```

Comment:
Let's keep only last bubble to simplify the code and reduce memory
```suggestion
  private _lastbubble?: MessageBubble;
```

Reply:
0e11dd012baad48fce62eeab905c60abc39aa9d5